### PR TITLE
feat(font): Switch to Work Sans and remove monospaced

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

- Added Work Sans via Google Fonts
- Removed usages of Space Mono
- Added `unicode-range` to currency_symbols @fontface

Completes #216

### Context of Change

By using `unicode-range` we can use currency_symbols at the top level and any characters that aren't  `u+e900` or `u+e901` fall through to the next font in the stack, in this case Work Sans.

### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### TypeScript/Hooks Update

- [ ] Updated files to React Hooks
- [ ] Updated files to TypeScript

### CODE REVIEW

File: src/containers/shared/css/global.scss [hunk 1]
```diff
@@ -14,30 +14,7 @@
...
-  font-weight: bold;
-  src: url('../fonts/SpaceMono-Bold.ttf');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  font-style: italic;
-  font-weight: bold;
-  src: url('../fonts/SpaceMono-BoldItalic.ttf');
+  unicode-range: u+e900-e901;
```
Comment (ledhed2222) [inline src/containers/shared/css/global.scss:17]: indentation looks off

Comment (ckniffen) [reply src/containers/shared/css/global.scss:17]: Optical illusion. https://github.com/ripple/explorer/blob/f07e7f39753ed245150643a4a21342fb3f6413a8/src/containers/shared/css/global.scss#L17

**Comment by shawnxie999** [comment]
just curious, why are we moving away from space mono?

File: src/containers/shared/css/global.scss (hunk 1 above)
Comment (ledhed2222) [reply src/containers/shared/css/global.scss:17]: add a comment in the CSS

Comment (ledhed2222) [reply src/containers/shared/css/global.scss:17]: also remove eot files

Comment (ledhed2222) [reply src/containers/shared/css/global.scss:17]: consider changing `font-family` to something super descriptive like `crypto_symbols`

**Comment by shawnxie999** [comment]
just curious, why are we moving away from space mono?

**Review by ledhed2222 — APPROVED** [review]

**Comment by ckniffen** [comment]
> just curious, why are we moving away from space mono?

That is the font used throughout the new designs. @bugsbunnies is this correct?

**Comment by bugsbunnies** [comment]
> > just curious, why are we moving away from space mono?
> 
> That is the font used throughout the new designs. @bugsbunnies is this correct?

@ckniffen Yes Work Sans will be the main font used throughout, but Space Mono will still be used for code snippets to pair with Work Sans when needed.

File: src/containers/shared/css/global.scss [hunk 2]
```diff
@@ -3,17 +3,17 @@
 @import './variables';
 @import './form';
 
-/* prettier-ignore */
+/**
+ * `currency_symbols` defines currencies fonts are missing, currently `u+e900` (XRP) and `u+e901` (BTC).
+ * By using unicode-range we specify `currency_symbols` at the top level and any other characters will fall through to
+ * the next font in the stack, in this case Work Sans.
```
Comment (ledhed2222) [inline src/containers/shared/css/global.scss:9]: super nit - i think this comment can be more descriptive and clear. the "falling through" that you mention is not defined here by the font-face itself or the use of `unicode-range`. perhaps something like

```text
`current_symbols` is used for currency codes that are missing in other font definitions, currently .....

By using unicode-range we specify which character points this font-family can be used for. When used in a style (and assuming this font family is first/takes precedent), only these characters will use this font-face. Anything else will fall through to the subsequent font-family.
```

**Review by shawnxie999 — APPROVED** [review]

File: src/containers/shared/css/global.scss [hunk 3]
```diff
@@ -3,41 +3,18 @@
...
-  src:
-    url('../fonts/currency_symbols.eot#iefix') format('embedded-opentype'),
-    url('../fonts/currency_symbols.ttf') format('truetype'),
-    url('../fonts/currency_symbols.woff') format('woff'),
-    url('../fonts/currency_symbols.svg') format('svg');
-}
-
-@font-face {
-  font-family: 'Space Mono';
-  src: url('../fonts/SpaceMono-Regular.ttf');
```
[the remark this answers is not in the document]
Comment (ckniffen) [reply src/containers/shared/css/global.scss:21]: I removed all instances.

**Pull request merged**